src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -2110,8 +2111,36 @@ function tooltipClinicalDataSection(
     }
 }
 
+function sampleIdForTooltip<D extends IPlotSampleData>(
+    d: D,
+    studiesInQuery: CancerStudy[]
```

Comment:
may be this should be a`Map` since it is lookup for study

Reply:
yeah I guess you're right, got lazy